pkgs/by-name/cm/cmake/package.nix

```diff
@@ -138,6 +138,8 @@ stdenv.mkDerivation (finalAttrs: {
     "CFLAGS=-D_FILE_OFFSET_BITS=64"
     "CXXFLAGS=-D_FILE_OFFSET_BITS=64"
   ]
+  # Workaround missing atomic ops with gcc <13
```

Comment:
Should we add a guard for GCC version?

Reply:
The workaround is not harmful when using gcc 13, just unnecessary. I'll remove it from cmake and everywhere else after we switch to gcc 13